app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -504,6 +510,11 @@ sealed class Event {
 }
 
 private fun Fact.toEvent(): Event? = when (Pair(component, item)) {
+    Component.FEATURE_PROMPTS to LoginDialogFacts.Items.DISPLAY -> Event.LoginDialogPromptDisplayed
+    Component.FEATURE_PROMPTS to LoginDialogFacts.Items.CANCEL -> Event.LoginDialogPromptCancelled
+    Component.FEATURE_PROMPTS to LoginDialogFacts.Items.NEVER_SAVE -> Event.LoginDialogPromptNeverSave
```

Comment:
So I just double checked where this fact is emitted and it will currently be emitted when a user presses "don't save" (ie refuse a login update) as well as "never save". We can fix this by just moving this up 1 line to where we are actually adding the exceptions.

https://github.com/mozilla-mobile/android-components/blob/master/components/feature/prompts/src/main/java/mozilla/components/feature/prompts/dialog/LoginDialogFragment.kt#L146

Reply:
ughhh rip. Thanks for letting me know. I'll put up a PR for AC!